frontend/src/data/nsf23313_2_1/dcids/values.ts

```diff
@@ -0,0 +1,129 @@
+    'Count_Student_Citizen_ScienceOrEngineeringOrHealth_MastersDegree_NotHispanicOrLatino_Male_Black',
+    'Count_Student_Citizen_ScienceOrEngineeringOrHealth_DoctorateDegree_NotHispanicOrLatino_Male_Black',
+    'Count_Person_Citizen_EducationalAttainmentDoctorateDegree_ScienceOrEngineeringOrHealth_NotHispanicOrLatino_Male_PostDoctoralAppointeeOccupation_Black',
+    'Count_Student_Citizen_ScienceOrEngineeringOrHealth_MastersDegreeOrDoctorateDegree_NotHispanicOrLatino_Male_NativeHawaiianOrOtherPacificIslanderAlone',
+    'Count_Student_Citizen_ScienceOrEngineeringOrHealth_MastersDegree_NotHispanicOrLatino_Male_NativeHawaiianOrOtherPacificIslanderAlone',
+    'Count_Student_Citizen_ScienceOrEngineeringOrHealth_DoctorateDegree_NotHispanicOrLatino_Male_NativeHawaiianOrOtherPacificIslanderAlone',
+    'Count_Person_Citizen_EducationalAttainmentDoctorateDegree_ScienceOrEngineeringOrHealth_NotHispanicOrLatino_Male_PostDoctoralAppointeeOccupation_NativeHawaiianOrOtherPacificIslanderAlone',
+    'Count_Student_Citizen_ScienceOrEngineeringOrHealth_MastersDegreeOrDoctorateDegree_NotHispanicOrLatino_Male_WhiteAlone',
+    'Count_Student_Citizen_ScienceOrEngineeringOrHealth_MastersDegree_NotHispanicOrLatino_Male_WhiteAlone',
+    'Count_Student_Citizen_ScienceOrEngineeringOrHealth_DoctorateDegree_NotHispanicOrLatino_Male_WhiteAlone',
+    'Count_Person_Citizen_EducationalAttainmentDoctorateDegree_ScienceOrEngineeringOrHealth_NotHispanicOrLatino_Male_PostDoctoralAppointeeOccupation_WhiteAlone',
+    'Count_Student_Citizen_ScienceOrEngineeringOrHealth_MastersDegreeOrDoctorateDegree_NotHispanicOrLatino_Male_TwoOrMoreRaces',
+    'Count_Student_Citizen_ScienceOrEngineeringOrHealth_MastersDegree_NotHispanicOrLatino_Male_TwoOrMoreRaces',
+    'Count_Student_Citizen_ScienceOrEngineeringOrHealth_DoctorateDegree_NotHispanicOrLatino_Male_TwoOrMoreRaces',
+    'Count_Person_Citizen_EducationalAttainmentDoctorateDegree_ScienceOrEngineeringOrHealth_NotHispanicOrLatino_Male_PostDoctoralAppointeeOccupation_TwoOrMoreRaces',
+    'Count_Student_Citizen_ScienceOrEngineeringOrHealth_MastersDegreeOrDoctorateDegree_UnknownEthnicity_Male_UnknownRace',
+    'Count_Student_Citizen_ScienceOrEngineeringOrHealth_MastersDegree_UnknownEthnicity_Male_UnknownRace',
+    'Count_Student_Citizen_ScienceOrEngineeringOrHealth_DoctorateDegree_UnknownEthnicity_Male_UnknownRace',
+    'Count_Person_Citizen_EducationalAttainmentDoctorateDegree_ScienceOrEngineeringOrHealth_UnknownEthnicity_Male_PostDoctoralAppointeeOccupation_UnknownRace',
+    'Count_Student_VisaHolder_ScienceOrEngineeringOrHealth_MastersDegreeOrDoctorateDegree_UnknownEthnicity_Male',
+    'Count_Student_VisaHolder_ScienceOrEngineeringOrHealth_MastersDegree_UnknownEthnicity_Male',
+    'Count_Student_VisaHolder_ScienceOrEngineeringOrHealth_DoctorateDegree_UnknownEthnicity_Male',
+    'Count_Person_VisaHolder_EducationalAttainmentDoctorateDegree_ScienceOrEngineeringOrHealth_UnknownEthnicity_Male_PostDoctoralAppointeeOccupation',
```

Comment:
It seems like we're missing Female Visaholders.

Reply:
added